packages/js/customer-effort-score/src/style.scss

```diff
@@ -107,8 +107,18 @@
 	}
 }
 
+.components-modal__content {
```

Comment:
Hi, @mdperez86! It looks like `.components-modal__content` is way too general, it's affecting all modals in admin.

![image](https://github.com/woocommerce/woocommerce/assets/3747241/6969ae4b-248c-4ac6-8662-e68c8833f3ae)

Originally reported by Vlad Olaru in this slack p1686918124623299/1686328973.559239-slack-C01SFMVEYAK.

Reply:
Hi @ilyasfoo, thank you for reporting that.
I created [this issue](https://github.com/woocommerce/woocommerce/issues/38748) and I'm working on a fix now.